apps/src/lib/ui/accounts/DeleteAccountDialog.story.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import DeleteAccountDialog from './DeleteAccountDialog';
+
+const PASSWORD = 'MY_PASSWORD';
+const DELETE_VERIFICATION = 'DELETE MY ACCOUNT';
+
+export default storybook => {
+  storybook
+    .storiesOf('DeleteAccountDialog', module)
```

Comment:
Please add to the Dialogs category ->   `.storiesOf('Dialogs/DeleteAccountDialog', module)

Reply:
Done!  Thanks.

Reply:
Good points raised!  Thanks !